src/views/3-Modeling/Modeling.vue

```diff
@@ -621,51 +626,50 @@
             id="legend-scale"
             transform="translate(0, 700)"
           >
-            <path
-              d="M228,643.5H807"
-              transform="translate(-28.24 -627.46)"
-              style="fill: none;stroke: white;stroke-miterlimit: 10;stroke-width: 3px"
-            />
+
             <text
-              class="text-annotate"
-              transform="translate(50 27.42)"
+              class="text-annotate left"
+              transform="translate(50 27)"
             >accurate</text>
             <text
-              class="text-annotate"
-              transform="translate(795 27.42)"
+              class="text-annotate right"
+              transform="translate(850 27)"
             >inaccurate</text>
-            <path
-              d="M247.5,656.5l-23-12,22-15"
-              transform="translate(-28.24 -627.46)"
-              style="fill: none;stroke: white;stroke-linecap: round;stroke-linejoin: round;stroke-width: 3px"
-            />
-            <path
-              d="M722.5,657.5l24-14-25-13"
-              transform="translate(30 -627.46)"
-              style="fill: none;stroke: white;stroke-linecap: round;stroke-linejoin: round;stroke-width: 3px"
-            />
+ 
           </g>
         </svg>
       </div>
     </figure>
     <!--     all the scrolling elements are created from modelingText.js content -->
     <article>
-      <div
-        v-for="text in text.methods"
-        :key="text.title"
-        class="step-container text-content"
-      >
+      <div id="scrollama-container">
+      <!-- create scrolling/sticky headers for each model section -->
+      <div 
+        v-for="(models, model_group) in text" 
+        :key="model_group" 
+        :class="model_group" 
+        class="step-container text-content">
         <div
-          :id="text.flubber_id"
-          class="step"
-        >
-          <h3> {{ text.title }} </h3>
-          <p> {{ text.method }} </p>
+          class="scroll-sticky">
+          <h2 >
+          {{ model_group }}
+          </h2>
         </div>
-      </div>
+        <!-- populate nested steps using text about each model -->
+        <div class="scrollama-steps">
+        <div class="step" 
+          v-for="model in models" 
+          :key="model" 
+          :id="model.flubber_id">
+          {{ model.method }}
+        </div></div>
+        </div>
+        </div>
+
     </article>
     <div id="map-container">
```

Comment:
ditto to what I mentioned above - will just need to move this to the end of chapter 2 at some point